Userland/Libraries/LibJS/Runtime/OrdinaryFunctionObject.cpp

```diff
@@ -99,8 +87,8 @@ void OrdinaryFunctionObject::initialize(GlobalObject& global_object)
         }
         define_property(vm.names.prototype, prototype, Attribute::Writable);
     }
-    define_native_property(vm.names.length, length_getter, {}, Attribute::Configurable);
-    define_native_property(vm.names.name, name_getter, {}, Attribute::Configurable);
+    define_property_or_throw(vm.names.length, { .value = Value(m_function_length), .configurable = true });
+    define_property_or_throw(vm.names.name, { .value = js_string(vm, m_name.is_null() ? "" : m_name), .configurable = true });
```

Comment:
Let's not omit `writable` and `enumerable` just because they're false, that means something different under certain circumstances.

```suggestion
    define_property_or_throw(vm.names.length, { .value = Value(m_function_length), .writable = false, .enumerable = false, .configurable = true });
    define_property_or_throw(vm.names.name, { .value = js_string(vm, m_name.is_null() ? "" : m_name), .writable = false, .enumerable = false, .configurable = true });
```

Reply:
my bad, fixed!